pkg/storage_nodes_count/storage_nodes_count.go

```diff
@@ -0,0 +1,112 @@
+package storagenodescount
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+const (
+	// SncYamlPath is the name of the yaml that will contain the decision matrix for storage nodes
+	SncYamlPath = "storage_nodes_count.yaml"
+	// YamlFilePath is the path where unit tests will find in this directory
+	YamlFilePath = "generator/" + SncYamlPath
```

Comment:
storage_nodes_count_test.go also needs this file. Is it okay if I temporarily save it here since we'll move to cloudops eventually?
I'll move it to bin/configs to be used from the container.